admin/controllers/questions.js

```diff
@@ -31,6 +32,12 @@ const getQuestions = async (req, res) => {
     await pupilLogonEventService.storeLogonEvent(null, schoolPin, pupilPin, false, 401, 'Unauthorised')
     return apiResponse.unauthorised(res)
   }
+
+  try {
+    await checkWindowService.isLoginAllowed()
```

Comment:
this will prevent anyone from using the system unless there is an active and current check window.

Can you think of any situation where this might be a problem?

Reply:
@GuyHarwood based on our discussion now separate errors are being thrown based on the scenario